tests/cases/bool/not.cairo

```diff
@@ -0,0 +1,8 @@
+fn main() -> (bool, bool, bool, bool) {
+    (
+        !! true,
+        !! false,
+        !! !! true,
+        !! !! false,
```

Comment:
Using `!!` would leave the value as-is. Seeing as the operator is duplicated below, maybe a single `!` was intended instead?
```suggestion
        !true,
        !false,
        !!true,
        !!false,
```